.github/workflows/tests.yml

```diff
@@ -0,0 +1,55 @@
+name: "Test Helm Charts"
+on:
+  - push
+  - pull_request
+
+env:
+  helm-version: v3.9.0
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.2.1
+
+      - name: Run chart-testing (lint)
+        run: ct lint --config ct.yaml
+
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        kube-version:
+          - "1.23"
+          - "1.24"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.2.1
+
+      - name: Start kuberenetes cluster with cert-manager
+        env:
+          KUBE_VERSION: ${{ matrix.kube-version }}
+        run: |
+          make cert-manager
+
+      - name: Wait for cluster to finish bootstraping
+        run: kubectl wait --for=condition=Ready pods --all --all-namespaces --timeout=300s
+
+      - name: Run chart-testing (install)
+        run: ct install --config ct.yaml --upgrade
```

Comment:
Still need to confirm this but there is a problem here. 

The `ct install` command, uninstalls the chart after everything is up and running and then waits for every resource to get deleted in the namespace it created. But some components don't go away even after a uninstall (like OTel Collector as it's a hook, and some secrets).

This means that `ct install` will get stuck and then timeout and fail, always.

One workaround is using a background job to delete the resources using kubectl once they are in up and running state.